sdk/digitaltwins/digitaltwins/swagger/README.md

```diff
@@ -0,0 +1,34 @@
+# KeyVault Keys Swagger Configuration
+
+> see https://aka.ms/autorest
+
+```yaml
+typescript:
+  package-name: "@azure/digitaltwins"
+  title: GeneratedClient
+  description: Digitaltwins Client
+use-extension:
+  "@autorest/typescript": "6.0.0-dev.20200727.1"
+azure-arm: true
```

Comment:
I feel like we should be able to turn this off?

Reply:
Yes, the latest autorest works without that! (yeah!)

---
In reply to: [470205397](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470205397) [](ancestors = 470205397)